file-formats/fugr/type/zif_fb_aff_func_v1.intf.abap

```diff
@@ -0,0 +1,238 @@
+  "! Processing type
+  "! $values {@link zif_fb_aff_func_v1.data:co_processing_type}
+  TYPES ty_processing_type TYPE c LENGTH 1.
+
+  CONSTANTS:
+    "! <p class="shorttext">Processing Type</p>
+    "! Processing type
+    BEGIN OF co_processing_type,
+      "! <p class="shorttext">Normal</p>
+      "! normal
+      normal TYPE ty_processing_type VALUE 'N',
+      "! <p class="shorttext">RFC</p>
+      "! rfc
+      rfc    TYPE ty_processing_type VALUE 'R',
+      "! <p class="shorttext">Update</p>
+      "! update
+      update TYPE ty_processing_type VALUE 'U',
+    END OF co_processing_type.
+
+
+  "! <p class="shorttext">RFC Scope</p>
+  "! Indicates the scope of function module calls
+  "! $values {@link zif_fb_aff_func_v1.data:co_rfc_scope}
+  TYPES ty_rfc_scope TYPE c LENGTH 1.
+
+  CONSTANTS:
+    "! <p class="shorttext">RFC Scope</p>
+    "! Indicates the scope of function module calls
+    BEGIN OF co_rfc_scope,
+      "! <p class="shorttext">From Same Client And User</p>
+      "! From same client and user
+      from_same_client_and_user TYPE ty_rfc_scope VALUE 'I',
+      "! <p class="shorttext">From Same System</p>
+      "! From same system
+      from_same_system          TYPE ty_rfc_scope VALUE 'C',
+      "! <p class="shorttext">From Any System</p>
+      "! From any system
+      from_any_system           TYPE ty_rfc_scope VALUE 'E',
+      "! <p class="shorttext">Not Classified</p>
+      "! Not classified
+      not_classified            TYPE ty_rfc_scope VALUE ' ',
+    END OF co_rfc_scope.
+
+  "! <p class="shorttext">RFC Version</p>
+  "! Indicates the permitted serializations for the function module
+  "! $values {@link zif_fb_aff_func_v1.data:co_rfc_version}
+  TYPES ty_rfc_version TYPE c LENGTH 10.
+
+  CONSTANTS:
+    "! <p class="shorttext">RFC Version</p>
+    "! Indicates the permitted serializations for the function module
+    BEGIN OF co_rfc_version,
+      "! <p class="shorttext">Fast Serialization Required</p>
+      "! Fast serialization required
+      fast_serialization_required TYPE ty_rfc_version VALUE '1',
+      "! <p class="shorttext">Any</p>
+      "! Any
+      any                         TYPE ty_rfc_version VALUE ' ',
+    END OF co_rfc_version.
+
+
+  TYPES:
+    "! <p class="shorttext">RFC specific fields</p>
+    "! Specific fields for rfc function modules
+    BEGIN OF ty_rfc_properties,
+      "! <p class="shorttext">BasXml Enabled</p>
+      "! Function module for classic RFC protocol and basXML protocol behaves the same from a semantic point of view
+      "! $required
+      basxml_enabled TYPE abap_bool,
+      "! <p class="shorttext">RFC Scope</p>
+      "! Indicates the scope of function module calls
+      "! $required
+      rfc_scope       TYPE ty_rfc_scope,
+      "! <p class="shorttext">RFC Version</p>
+      "! Indicates the permitted serializations for the function module
+      "! $required
+      rfc_vers        TYPE ty_rfc_version,
```

Comment:
Sorry, for being picky. ;) Can't we change it to `rfc_version`?
```suggestion
      rfc_version        TYPE ty_rfc_version,
```

Reply:
Yes, we can :)